### PR TITLE
Update pair-programming.md

### PR DESCRIPTION
**Pull request by madalynrose**
## Description

update process to reflect our new streaming process and point to our youtube channel with previous streams

### CODE REVIEW

File: docs/contributing/pair-programming.md [hunk 1]
```diff
@@ -2,16 +2,19 @@
 title: Open Community Pair Programming Sessions
 ---
 
-The best part of open source is the community, and every community is stronger when it works together. To help build the strongest community possible, Gatsby is offering free [45 minute pair programming sessions][cal] to anyone and everyone in the open source community.
+The best part of open source is the community, and every community is stronger when it works together. To help build the strongest community possible, Gatsby is offering free [45 minute pair programming sessions][form] to anyone and everyone in the open source community.
 
 ## How community pair programming sessions work
 
 1.  Select or create [an issue](https://github.com/gatsbyjs/gatsby/issues) you would like to work on and work on it yourself
```
Comment (laurieontech) [inline docs/contributing/pair-programming.md]:
```suggestion
1.  Select or create [an issue](https://github.com/gatsbyjs/gatsby/issues) you would like to work on and work on it yourself.
```

File: docs/contributing/pair-programming.md [hunk 2]
```diff
@@ -2,16 +2,19 @@
...
 2.  [Create a pull request](https://www.gatsbyjs.org/contributing/how-to-open-a-pull-request/) when you encounter any problem
```
Comment (laurieontech) [inline docs/contributing/pair-programming.md]:
```suggestion
2.  [Create a pull request](https://www.gatsbyjs.org/contributing/how-to-open-a-pull-request/) when you encounter any problem.
```

File: docs/contributing/pair-programming.md [hunk 3]
```diff
@@ -2,16 +2,19 @@
...
-3.  Sign up for any available slot on [the open pairing calendar][cal]
-4.  You’ll be paired with a Gatsby team member
-5.  Join the video meeting during your time slot (we use [Zoom](https://zoom.us) for meetings)
-6.  We'll help you with your pull request or issue!
+3.  Fill in your details in [the pair programming request form][form]
```
Comment (laurieontech) [inline docs/contributing/pair-programming.md]:
```suggestion
3.  Fill in your details in [the pair programming request form][form].
```

**Comment by madalynrose** [comment]
Thanks @laurieontech! I applied your suggestions. I didn't even notice I left off punctuation so thank you for catching that 🙈

**Comment by laurieontech** [comment]
Looks great!

**Pull request merged**